src/riscv/lib/src/jit/builder/control_flow_graph.rs

```diff
@@ -136,6 +136,30 @@ impl<'info, T> BudgetCheckLoc<'info, T> {
     }
 }
 
+/// Information about a step budget check.
+///
+/// A budget is the maximum number of steps that can be taken from this point
+/// without exceeding the overall step budget for the sequence.
+#[derive(Debug)]
+pub struct BudgetCheck<'info, T> {
+    /// Budget associated with this budget check.
```

Comment:
```suggestion
    /// Number of steps that may be run from this point, before another budget check takes place.
```

nit? but otherwise this documentation kind of doesn't really say anything

Reply:
done [here](https://github.com/tezos/riscv-pvm/compare/384cb0f79c3509deacb5187954c060f04c37092d..5bc1523347888a5c69e5f818e817e18e5d9c2543)